src/app/pages/about/about.component.scss

```diff
@@ -1,9 +1,14 @@
-@import '../../style/_variables.scss' ,'../../style/_animation.scss';
+@use '../../style/_variables' as *  ;
+@use '../../style/_animation'  as *;
 h1{
```

Comment:
please use class with the parent component to wrap all the subjects inside it (can lead to css conflicts)